server/routes/organizations/employees/employee-controller.js

```diff
@@ -1,75 +1,79 @@
-var models = require('../../../db/models');
-var User = models.User;
+module.exports = function(app){
```

Comment:
This is an awesome refactor/fix. Great Job! This will avoid unnecessary connections to our db